addons/l10n_eu_service/tests/test_oss.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+
+from odoo.addons.l10n_eu_service.models.eu_tag_map import EU_TAG_MAP
+from odoo.addons.account.tests.account_test_savepoint import AccountTestInvoicingCommon
+from odoo.tests import tagged
+
+
+@tagged('post_install', '-at_install')
+class TestOSSBelgium(AccountTestInvoicingCommon):
+
+    @classmethod
+    def setUpClass(self, chart_template_ref='l10n_be.l10nbe_chart_template'):
+        try:
+            super().setUpClass(chart_template_ref=chart_template_ref)
+        except ValueError as e:
+            if e.args[0] == "External ID not found in the system: l10n_be.l10nbe_chart_template":
+                self.skipTest(self, reason="Belgian CoA is required for this testSuite but l10n_be isn't installed")
+            else:
+                raise e
+        self.company_data['company'].country_id = self.env.ref('base.be')
+        self.company_data['company']._map_eu_taxes()
+
+    def test_country_tag_from_belgium(self):
+        # get an eu country which isn't the current one:
+        another_eu_country_code = (self.env.ref('base.europe').country_ids - self.company_data['company'].country_id)[0].code
+        tax_oss = self.env['account.tax'].search([('name', 'ilike', f'%{another_eu_country_code}%')], limit=1)
+
+        invoice_tax_tag_ids = tax_oss.invoice_repartition_line_ids.filtered(lambda x: x.repartition_type == 'base').tag_ids
+        tag_base_exempted_47 = self.env.ref("l10n_be.tax_report_line_47").tag_ids.filtered(lambda t: not t.tax_negate)
+        self.assertTrue(tag_base_exempted_47 in invoice_tax_tag_ids, "Invoice tag from Belgium CoA not correctly linked")
+
+        refund_tax_tag_ids = tax_oss.refund_repartition_line_ids.filtered(lambda x: x.repartition_type == 'base').tag_ids
+        tag_base_exempted_49 = self.env.ref("l10n_be.tax_report_line_49").tag_ids.filtered(lambda t: not t.tax_negate)
+        self.assertTrue(tag_base_exempted_49 in refund_tax_tag_ids, "Refund tag from Belgium CoA not correctly linked")
```

Comment:
discussed with WAN:
Ok.
Not blocking: could look better by splitting a bit vertically.